assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -1,5 +1,6 @@
 export const CHATABLE_TYPES = {
   directMessageChannel: "DirectMessageChannel",
   topicChannel: "Topic",
+  tagChannel: "Tag",
```

Comment:
I recall @markvanlan saying that tagChannels will be gone soon.

Reply:
Yes I think you are rigjt. I guess we can remove this when they are actually gone.